IntegrationTests/Assets/ExternalDependencyManager.meta

```diff
@@ -0,0 +1,8 @@
+fileFormatVersion: 2
```

Comment:
Is the `ExternalDependencyManager` folder is not in the repo so I don't think we need this file